src/workloads/query/multiplanner/CompoundIndexes.yml

```diff
@@ -278,10 +284,70 @@ Actors:
       collection: *coll
       query: &query {
         Filter: {
-          tenantId: {$eq : 0},
+          tenantId: {$eq : 1},
```

Comment:
~~I was wondering why the tenantId changed but you are still picking the tenantId.~~

~~Is the overall effect of the query change to roughly retain the selectivity of the original?~~

You have answered this in the description.

Reply:
The particular change was because I wanted to have a constant above which determines the number if unique `tenantIds`, which in this case is 3. The `{distribution: uniform, min: x, max: y}` expects the min and max bounds to be inclusive, so it's easiest to express using a min of 1 rather than 0. That meant I had to update the filter, since we used to generate tenant ids starting at 0.